### DISCUSSION TITLE
Seattle Fire Station Locator

### DISCUSSION THREAD

**Discussion by Subarashiihibi** [jakobzhao/geog495#52] [Final Project]
Description 
Application： https://subarashiihibi.github.io/firestation_locator/index.html
Repo： https://subarashiihibi.github.io/firestation_locator/
As the fire has always been an issue to Seattle, Washington, and inspired by the store locator, I am constructing the fire station locator map. Just like a number of the lab assignments we have done before, the web application incorporates geocoding, Map box APIs, as well as turf.js for distance calculating. The project is still under construction and hopefully it could reach the goal that the when typing the location in the search bar, it could return the closest fire station in route mile with camera zoom in and pop-up. (Currently I am running using internal geojson file with smaller size but will change it later).  If you have advice on doing the project I would be appreciative to learn it. Thanks!
![image](https://user-images.githubusercontent.com/80044018/145507212-4ebc7893-4b62-4745-b9fb-ec79ef0b1951.png)

**Comment by zeyany2** [comment]
Hi! It is actually interesting that we both choose to do the application over Seattle fire stations. It is cool you have a search bar that I didn't cover in my application. I might need to think about it. For your data, I suggest you find seattle fire stations geojson from Seattle GeoData.

**Comment by aembar** [comment]
I really like the basemap for this application as we are able to clearly see the different roads/streets. I also like that you implemented a closing button for the popups. Some suggestions I have are ordering the fire stations in descending/ascending order(as of now they seem to be randomized). Also maybe change the icon of the fire stations to include red/orange/yellow colors.

**Comment by NahomAgidew** [comment]
Really good work. I like the custom icons 🔥 you made. My suggestion is more about the code. I think it would be better if you separated the CSS and JS code block into their own files.
